Lib.Web.Mvc/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="JetBrains.Annotations" version="10.0.0" targetFramework="net40" />
```

Comment:
Applying two annotations is not good enough reason for dragging new external dependency.